lib/invoicing/currency_value.rb

```diff
@@ -119,7 +119,7 @@ def acts_as_currency_value(*args)
 
     included do
       # Register callback if this is the first time acts_as_currency_value has been called
-      before_save :write_back_currency_values, :if => "currency_value_class_info.previous_info.nil?"
+      before_save :write_back_currency_values
```

Comment:
Why is this `if` condition removed @Atul9 ?

Reply:
Use a proc or a bloc instead?
`if: { currency_value_class_info.previous_info.nil? }`

Reply:
done, I have updated the PR to use `if: currency_value_class_info.previous_info.nil?`